src/rust/dbv2/src/models.rs

```diff
@@ -50,16 +50,16 @@ pub struct PlaceLimitOrderEvent {
     pub event_idx: BigDecimal,
     pub time: DateTime<Utc>,
     pub market_id: BigDecimal,
-    pub maker_address: String,
-    pub maker_custodian_id: BigDecimal,
-    pub maker_order_id: BigDecimal,
-    pub maker_side: bool,
-    pub integrator_address: String,
+    pub user: String,
```

Comment:
You're going to need to update the processor to accommodate the new names.

Reply:
Yes, that will come in a pr on the other repo.